app/src/main/java/movierating/services/AverageReviewScoreForMovie.java

```diff
@@ -0,0 +1,26 @@
+package movierating.services;
+
+import movierating.models.Movie;
+import movierating.repositories.MovieRepository;
+
+public class AverageReviewScoreForMovie {
```

Comment:
Aren't there any test cases to test these in the main?

Reply:
Have added few test cases.